### DISCUSSION TITLE
Installation finishes but there is an error

### DISCUSSION THREAD

**Discussion by TheRedM** [MusicPlayerDaemon/MPD#1404] [Q&A]
After running
```sudo apt-get install mpd```
I get this error at the end of the installation

```Created symlink /etc/systemd/user/default.target.wants/mpd.service → /usr/lib/systemd/user/mpd.service.
Created symlink /etc/systemd/user/sockets.target.wants/mpd.socket → /usr/lib/systemd/user/mpd.socket.
Created symlink /etc/systemd/system/multi-user.target.wants/mpd.service → /lib/systemd/system/mpd.service.
Created symlink /etc/systemd/system/sockets.target.wants/mpd.socket → /lib/systemd/system/mpd.socket.
Job failed. See "journalctl -xe" for details.
Processing triggers for libc-bin (2.31-0ubuntu9.2) ...
Processing triggers for systemd (245.4-4ubuntu3.14) ...
Processing triggers for man-db (2.9.1-1) ...
Processing triggers for hicolor-icon-theme (0.17-2) ...
```
Running ```journalctl -xe``` i found multiple errors about `mpd.socket`

The first error there says

```
systemd[1]: mpd.socket: Socket service mpd.service already active, refusing.
systemd[1]: Failed to listen on mpd.socket
```

I don't know what's causing this problem.

**Comment by leigh123linux** [comment]
Ubuntu needs to fix their crappy packaging issue.

**Reply by TheRedM** [reply]
> Ubuntu needs to fix their crappy packaging issue.

Do you happen to know a fix for this error? i tried everything i know

**Comment by leigh123linux** [comment]
The ubuntu package creates mpd.socket for user and system, you can't run it twice.

**Reply by TheRedM** [reply]
So there is no solution! Running `remove` and `purge` and reinstalling mpd didn't fix it, always same error, it's kinda frustrating facing this issue during the installation process, it's a fresh Ubuntu installation that's why is hard to blame it on interference, i hope it get fixed in the future.

**Comment by leigh123linux** [comment]
Did you file the issue at ubuntu bug tracker (launchpad)?

**Reply by TheRedM** [reply]
No i didn't because i thought that was a bug from mpd side, i'll report the issue now.
